public/app/features/alerting/unified/components/mute-timings/useMuteTimings.tsx

```diff
@@ -140,40 +143,24 @@ export const useMuteTimings = ({ alertmanager }: BaseAlertmanagerArgs) => {
 export const useCreateMuteTiming = ({ alertmanager }: BaseAlertmanagerArgs) => {
   const useK8sApi = shouldUseK8sApi(alertmanager);
 
-  const dispatch = useDispatch();
   const [createGrafanaTimeInterval] = useCreateNamespacedTimeIntervalMutation();
-  const [getAlertmanagerConfig] = useLazyGetAlertmanagerConfigurationQuery();
-
-  const isGrafanaAm = alertmanager === GRAFANA_RULES_SOURCE_NAME;
+  const [updateConfiguration] = useProduceNewAlertmanagerConfiguration();
 
-  if (useK8sApi) {
+  const addToK8sAPI = useAsync((interval: MuteTimeInterval) => {
```

Comment:
Can we swap this back to be `{ interval }` as the argument? So its consistent with the `update` method and gives us scope to add more arguments inline with RTKQ's approach?
(And maybe we have a helper type for the repeated `{ interval }: { interval: MuteTimeInterval }` definition?)